src/app/tests/suites/TestCluster.yaml

```diff
@@ -2894,7 +2894,7 @@ tests:
       command: "subscribeAttribute"
       attribute: "list_int8u"
       minInterval: 2
-      maxInterval: 10
+      maxInterval: 5
```

Comment:
The other option is to set a large-enough `stepTimeoutSecs` here.  Would that be preferable to just bumping `kTimeoutInSeconds` globally?

Reply:
That would be fine too - I do not have personal preferences and I forgot about `stepTimeoutSecs` fwiw ;)
I also have to say that `stepTimeoutSecs` and `kTimeoutInSeconds` will both disappears with https://github.com/vivien-apple/connectedhomeip-1/commit/fd150b23357a4932a3f1db263e8681af1f6a8947